source/sp.rst

```diff
@@ -485,3 +485,42 @@ Example: BF\ :sub:`3`\
       4 F    -0.233   -0.335   -0.284
 
 The Fukui indexes for BF\ :sub:`3`\  indicate the most negative f(+) value and a positive value for f(-) at the boron atom. Thus, a nucleophilic attack can be expected at the boron atom.
+
+H\ :sub:`0`\  Tuning
+====================
+
+For special cases it can be beneficial to tune the H\ :sub:`0`\  Hamiltonian by modifying the atom-pairwise parameters. In order to do this, atom pairs need to be added to the ``$pairpar`` block in the parameter file ``$XTBPATH/param_gfn2-xtb.txt``.
```

Comment:
Should we really recommend to modify the global GFN2-xTB parameter path here? I would prefer that we encourage the creation of a copy with a meaningful name that is than further modified.